include/LDtkLoader/Layer.hpp

```diff
@@ -57,6 +57,7 @@ namespace ldtk {
         void updateTileVerticesCol(const Tile& tile) const;
 
         const LayerDef* m_definition = nullptr;
+        const Tileset* m_overrideTileset = nullptr;
```

Comment:
```suggestion
        const Tileset* m_override_tileset = nullptr;
```
Just to be consistent with variable naming